### PR TITLE
ENH: Upgrade visualizer to use q2templates

### PR DESCRIPTION
**Pull request by jakereps**
Upgrades example visualizer to utilize `q2templates`.

### CODE REVIEW

File: {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/_dummy_visualizer.py [hunk 1]
```diff
@@ -2,48 +2,30 @@
...
-        fh.write('</head>\n')
-        fh.write('<body>\n')
-        fh.write('<h3>mapping1:</h3>\n')
-        fh.write(df1.to_html(index=False, classes='dummy-class'))
-        fh.write('<h3>mapping2:</h3>\n')
-        fh.write(df2.to_html(index=False, classes='dummy-class'))
-        fh.write('</body></html>')
+    index = os.path.join(TEMPLATES, 'index.html')
+    context = {
+        "mapping1": df1.to_html(index=False, classes='dummy-class'),
```
Comment (thermokarst) [inline {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/_dummy_visualizer.py]: Most/all of the other visualizations with a table included have something along the following lines, in order to format the HTML table correctly:

``` python
table = d1.to_html(index=False, classes="table table-striped table-hover")
table = table.replace('border="1"', 'border="0"')
```

Can you add that for both of the tables in this viz?

File: {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+
+{% block head %}
```
Comment (thermokarst) [inline {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html]: I think we can drop all of this `{% block head %}` section once the table is bootstrappin

File: {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    .dummy-class th, td {
+        padding: 8px;
+        text-align: left;
+        border-bottom: 1px solid #ddd;
+    }
+</style>
+{% endblock %}
+
+{% block content %}
+    <h3>mapping1:</h3>
```
Comment (thermokarst) [inline {{cookiecutter.distribution_name}}/{{cookiecutter.package_name}}/assets/index.html:7]: Can you stick these into a two-col grid layout? That way folks who might not be familiar with bootstrap can see that feature in action.

**Review by thermokarst — CHANGES_REQUESTED** [review]
Thanks @jakereps! A few minor comments and then this is GTG

**Comment by jakereps** [comment]
Thanks @thermokarst, should be good to go 👍 !

**Review by thermokarst — APPROVED** [review]

**Pull request merged**